Test/DependencyCollector/FunctionalTests/TestApps/AspxCore/Controllers/ExternalCallsController.cs

```diff
@@ -0,0 +1,127 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Net.Http;
+using System.Text;
+
+namespace AspxCore.Controllers
+{
+    [Route("external/calls")]
+    public class ExternalCallsController : Controller
+    {/// <summary>
+     /// Invalid Hostname to trigger exception being thrown
+     /// </summary>
+        private const string InvalidHostName = "http://www.zzkaodkoakdahdjghejajdnad.com";
+
+        private string GetQueryValue(string valueKey)
+        {
+            return Request.Query[valueKey].ToString();
+        }
+
+        // GET external/calls
+        [HttpGet]
+        public string Get()
+        {
+            string title = "(No title set)";
+            string response = "(No response set)";
+
+            string type = GetQueryValue("type");
+            string countStr = GetQueryValue("count");
+
+            int count;
+            if (!int.TryParse(countStr, out count))
+            {
+                count = 1;
+            }
+
+            switch (type)
+            {
+                case "http":
+                    title = "Made Sync GET HTTP call to bing";
+                    response = MakeHttpGetCallSync(count, "bing");
+                    break;
+                case "httppost":
+                    title = "Made Sync POST HTTP call to bing";
+                    response = MakeHttpPostCallSync(count, "bing");
+                    break;
+                case "failedhttp":
+                    title = "Made failing Sync GET HTTP call to bing";
+                    response = MakeHttpCallSyncFailed(count);
+                    break;
+                default:
+                    title = $"Unrecognized request type '{type}'";
+                    response = "";
+                    break;
+            }
+
+            return $"<HTML><BODY>{title}<BR/>{response}</BODY></HTML>";
+        }
+
+        /// <summary>
+        /// Make sync http GET calls
+        /// </summary>        
+        /// <param name="count">no of GET calls to be made</param>        
+        /// <param name="hostname">the GET call will be made to http://www.hostname.com</param>        
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2202:Do not dispose objects multiple times", Justification = "Reviewed manually")]
```

Comment:
Where is this rule being triggered?  I don't see any using statements or explicit Dispose calls...